src/main/mw.c

```diff
@@ -419,10 +420,23 @@ void processRx(void)
     }
 
     throttleStatus_e throttleStatus = calculateThrottleStatus(&masterConfig.rxConfig, masterConfig.flight3DConfig.deadband3d_throttle);
+    rollPitchStatus_e rollPitchStatus =  calculateRollPitchCenterStatus(&masterConfig.rxConfig);
 
+    /* In airmode Iterm should be prevented to grow when Low thottle and Roll + Pitch Centered. */
+    /* This is needed to prevent Iterm winding on the ground, but keep full stabilisation on 0 throttl in air */
```

Comment:
But antiWindupProtection  is enabled when on zero throttle and sticks centered in air.
Limit is taken as value I-term just before activating antiWindupProtection, so it may be very small.
This will probably prevent wind gust rejection when gliding ..